projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -337,5 +356,6 @@ const enum ExplorerQueryParam {
   Group = 'group',
   OtherGroup = 'other',
   GroupLimit = 'limit',
-  Series = 'series'
+  Series = 'series',
+  Sort = 'sort'
```

Comment:
Currently is not generating an issue related to this new query param. Data is getting fine from backend

Reply:
Because this isn't being used - https://github.com/hypertrace/hypertrace-ui/pull/1747#discussion_r971062922 . Take a look at how the other query params are being written and read from the url.

Reply:
I've already added the query params read and write actions. Does it all or do I need to add something extra for the table?